sklearn/tests/test_docstring_parameters.py

```diff
@@ -75,8 +61,6 @@ def test_docstring_parameters():
 
     incorrect = []
     for name in PUBLIC_MODULES:
-        if name.startswith('_') or name.split(".")[1] in IGNORED_MODULES:
```

Comment:
How come we no longer need the _ condition?

Reply:
All names start by `sklearn.` anyway, and

```python
PUBLIC_MODULES = set([pckg[1] for pckg in walk_packages(prefix='sklearn.',        
                                                        path=sklearn.__path__) 
                      if not ("._" in pckg[1] or ".tests." in pckg[1])]) 
```